tools/python/boutiques/searcher.py

```diff
@@ -121,7 +121,9 @@ def parse_basic_info(self, hit):
         id = "zenodo." + str(hit["id"])
         title = hit["metadata"]["title"]
         description = hit["metadata"]["description"]
-        downloads = hit["stats"]["version_downloads"]
+        downloads = 0
+        if "version_downloads" in hit["stats"]:
+            downloads = hit["stats"]["version_downloads"]
```

Comment:
Not a big deal but it would be more Pythonic to write `downloads = hit[..][...] if "version_downloads" in hit["stats"] else 0`.